chain/client/tests/process_blocks.rs

```diff
@@ -851,7 +851,8 @@ fn test_gc_with_epoch_length_common(epoch_length: NumBlocks) {
         blocks.push(block);
     }
     for i in 1..=epoch_length * (NUM_EPOCHS_TO_KEEP_STORE_DATA + 1) {
-        if i <= epoch_length {
+        println!("height = {}", i);
```

Comment:
Remove `println`

Reply:
This is in a test, so I think it is fine.